### PR TITLE
Update loop conditions to be more idiomatic; fix missing imports

### PR DESCRIPTION
**Pull request by AndreyBozhko**
#### What does this implement/fix?
This replaces the loop conditions `while 1:` with more idiomatic `while True:`, and fixes a couple of missing imports.

#### CheckList
- [ ] Did you add an unittest for your new example or defect fix?
- [ ] Did you add documents for your new example?
- [ ] All CIs are green? (You can check it after submitting)

### CODE REVIEW

File: PathPlanning/RRT/rrt_with_sobol_sampler.py [hunk 1]
```diff
@@ -30,6 +30,7 @@
 import random
 import matplotlib.pyplot as plt
 import numpy as np
+import sys
```
Comment (AtsushiSakai) [inline PathPlanning/RRT/rrt_with_sobol_sampler.py:33]: Thank you for your PR. Is it possible to add a test to run this script?

Comment (AtsushiSakai) [reply PathPlanning/RRT/rrt_with_sobol_sampler.py:33]: Ignore this comment and please merge latest master branch. Another PR fixed this.

Comment (AndreyBozhko) [reply PathPlanning/RRT/rrt_with_sobol_sampler.py]: Thanks - I rebased these changes on top of the latest master branch.

**Review by AtsushiSakai — APPROVED** [review]
LGTM. Thanks.

**Pull request merged**